app/dashboard/views/support.py

```diff
@@ -0,0 +1,93 @@
+import json
+import urllib.parse
+from typing import Union
+
+import requests
+from flask import render_template, request, flash, url_for, redirect
+from flask_login import login_required, current_user
+from werkzeug.datastructures import FileStorage
+
+from app.dashboard.base import dashboard_bp
+from app.log import LOG
+from app.models import Mailbox
+from app.config import ZENDESK_HOST
+
+VALID_MIME_TYPES = ['text/plain', 'message/rfc822']
+
+
+@dashboard_bp.route("/support", methods=["GET"])
+@login_required
+def show_support_dialog():
+    mailbox = Mailbox.get(current_user.default_mailbox_id)
+    return render_template("dashboard/support.html", ticketEmail=mailbox.email)
+
+
+def upload_file_to_zendesk(file: FileStorage) -> Union[None, str]:
+    if file.mimetype not in VALID_MIME_TYPES and not file.mimetype.startswith('image/'):
+        flash('File {} is not an image, text or an email'.format(file.filename), "warning")
+        return None
+    escaped_filename = urllib.parse.urlencode({'filename': file.filename})
+    url = 'https://{}/api/v2/uploads?{}'.format(ZENDESK_HOST, escaped_filename)
+    headers = {'content-type': file.mimetype}
+    response = requests.post(url, headers=headers, data=file.stream)
+    if response.status_code != 201:
+        if response.status_code == 401 or 422:
+            LOG.debug('Could not authenticate')
+            return None
```

Comment:
Nits: we can just call `return` (without `None`) as None is the default return value for a function in Python.